cloudinit/net/tests/test_network_state.py

```diff
@@ -2,12 +2,62 @@
 
 from unittest import mock
 
+import pytest
+
+from cloudinit import safeyaml
 from cloudinit.net import network_state
 from cloudinit.tests.helpers import CiTestCase
 
 netstate_path = 'cloudinit.net.network_state'
 
 
+_V1_CONFIG_NAMESERVERS = """\
+network:
+  version: 1
+  config:
+    - type: nameserver
+      interface: {}
```

Comment:
Can we specifically name this format variable or give it an index so it's more clear that we will probably be using this as a format string. Initial glance at the string made me think we were expecting a dict type in tests.
```suggestion
      interface: {0}     # or {int_name}
```